src/bundle/Form/Locations/Ordering.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUiBundle\Form\Locations;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\API\Repository\Values\Content\LocationUpdateStruct;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Translation\Translator;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class Ordering extends AbstractType
+{
+    /**
+     * @var Translator
+     */
+    private $translator;
+
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('Submit', SubmitType::class)
+            ->add(
+                'sortField',
+                ChoiceType::class,
+                [
+                    'choices' => $this->getSortFields($options['current_sort_field']),
+                    'required' => true,
+                ]
+            )
+            ->add(
+                'sortOrder',
+                ChoiceType::class,
+                [
+                    'choices' => $this->getSortOrders(),
+                    'required' => true,
+                ]
+            );
+    }
+
+    /**
+     * @return array
+     */
+    protected function getSortOrders()
+    {
+        return [
+            /** @Desc("Ascending") */
+            $this->translator->trans('locationview.details.ascending', [], 'locationview') => Location::SORT_ORDER_ASC,
+            /** @Desc("Descending") */
+            $this->translator->trans('locationview.details.descending', [], 'locationview') => Location::SORT_ORDER_DESC,
+        ];
+    }
+
+    /**
+     * @param int|null $currentSortField
+     * @return array
+     */
+    protected function getSortFields($currentSortField)
+    {
+        $defaultSortFields = [
```

Comment:
I'd just call that `sortField`. `default` in this context doesn't really make sense.